feathr_project/feathr/datasets/nyc_taxi.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from threading import local
+from urllib.parse import urlparse
+
+import pandas as pd
+from pyspark.sql import DataFrame, SparkSession
+
+from feathr.datasets.utils import maybe_download
+from feathr.utils.platform import is_databricks
+
+
+NYC_TAXI_SMALL_URL = (
```

Comment:
Shall we move it to `constants.py` or new a `datasets/constants.py`? It will be useful if we extend to more datasets.